src/main/java/openfasttrack/report/PlainTextReport.java

```diff
@@ -140,23 +147,10 @@ private void renderFailureIds(final PrintStream report)
     private void renderFailureDetails(final PrintStream report)
     {
         this.trace.getUncoveredItems().stream() //
-                .sorted((item, other) -> item.getId().compareTo(other.getId())) //
```

Comment:
Could be simplified and more explicit by using [Comparator.comparing()](https://docs.oracle.com/javase/8/docs/api/java/util/Comparator.html#comparing-java.util.function.Function-)